latex/epoch.tex

```diff
@@ -0,0 +1,54 @@
+The pool reaping in rules are discribed in
+Figure~\ref{fig:ts-types:pool-reap} and Figure~\ref{fig:rules:pool-reap}.
+
+%%
+%% Figure - Pool Reap
+%%
+
+
+\begin{figure}
+  \emph{Pool Reap Transition}
+  \begin{equation*}
+    \_ \vdash \_ \trans{poolreap}{} \_ \in
+    \powerset (\Slot \times \PState \times \PState)
+  \end{equation*}
+  %
+  \caption{Pool Reap Transition}
+  \label{fig:ts-types:pool-reap}
+\end{figure}
+
+
+%%
+%% Figure - Pool Rules
+%%
+\begin{figure}
+
+  \begin{equation}\label{eq:pool-reap}
+    \inference[Pool-Reap]
+    {
+      \var{retired} = \var{retiring}^{-1}~\var{(\epoch{slot})}
+      & \var{retired} \neq \emptyset
```

Comment:
is this needed ? Else the rule simply would not have an observable effect, right?

Reply:
yes, that's correct.  It was suggested that I add that requirement in a previous code review, but honestly I'm happy either way.